client/webserver/site/src/js/app.ts

```diff
@@ -807,31 +807,20 @@ export default class Application {
    * haveActiveOrders returns whether or not there are active orders involving a
    * certain asset.
    */
-  haveAssetOrders (assetID: number): boolean {
+  haveActiveOrders (assetID: number): boolean {
     for (const xc of Object.values(this.user.exchanges)) {
       if (!xc.markets) continue
       for (const market of Object.values(xc.markets)) {
         if (!market.orders) continue
         for (const ord of market.orders) {
           if ((ord.baseID === assetID || ord.quoteID === assetID) &&
-            (ord.status < StatusExecuted || hasLiveMatches(ord))) return true
+            (ord.status < StatusExecuted || hasActiveMatches(ord))) return true
         }
       }
     }
     return false
   }
 
-  walletIsActive (assetID: number): boolean {
-    if (this.haveAssetOrders(assetID)) return true
-    for (const xc of Object.values(this.user.exchanges)) {
-      if (!xc) continue
-      if (xc.pendingFee && xc.pendingFee.assetID === assetID) {
-        return true
-      }
-    }
-    return false
-  }
```

Comment:
This **func** doesn't seem to be used, hence removed.

Reply:
I'd be surprised if we don't need this back but with a bonds check instead of the pending fee.  Can re-create though.